packages/components/pie-thumbnail/src/thumbnail.scss

```diff
@@ -3,6 +3,7 @@
   --thumbnail-border-radius: var(--dt-radius-rounded-b);
   --thumbnail-bg-color: var(--dt-color-container-default);
   --thumbnail-border-color: transparent;
+  --thumbnail-padding: calc(var(--dt-spacing-a) / 2);
```

Comment:
More feedback to design – should we have a token for `2px` if we're going to be using it in components as a spacing value?

cc @RLumleyJET

Reply:
i will raise this with designers on Moday's standup